unfoldingWord/translationCore
# Bug fix/mcleanb/2715/windows import fix (QA fail)

#### This pull request addresses:

A Fix for Windows so that the local import file dialog stays modal: background is dimmed and cannot click off file dialog.

- had to change to use ipcRenderer.sendSync() for dialog to be modal on windows.
- had to put dim screen call before toggle FABs to get screen to go dark on windows.

#### How to test this pull request:

On Windows select import a local project, the main screen should be dimmed, but when you click on the FAB, the file Explorer should stay in front.  Verify that import cancel still works, and then verify that file import still works.
On Mac should see same behavior as above.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3201)
